server/src/main/java/org/elasticsearch/common/settings/SettingsModule.java

```diff
@@ -215,6 +231,10 @@ public ClusterSettings getClusterSettings() {
         return clusterSettings;
     }
 
+    public Set<Setting<?>> getConsistentSecureSettings() {
```

Comment:
I think this is leaking a detail that consistent settings can only be secure settings. I think `getConsistentSettings` is good

Reply:
I don't see it as an unimportant detail, but acted on the suggestion.